lib/rack/cache/meta_store.rb

```diff
@@ -81,7 +81,11 @@ def store(request, response, entity_store)
       headers.delete 'Age'
 
       entries.unshift [stored_env, headers]
-      write key, entries
+      if request.env['rack-cache.use_native_ttl'] && response.fresh?
```

Comment:
would this be better as an option when creating the store ?
this is supposed to be passed in from the outside ... or via another middleware ?